.changeset/thirty-toys-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+"@orbit-ui/transition-components": major
+---
+
+Alert UI fixes - Alert is now closable by default
```

Comment:
Alert is now closable by default. This is potentially a breaking change if you we're not expecting the alert to be closable. 
To fix this, either add `dismissable={false}` to the Alert, or handle the `onClose` event handler